platform/engineering/frontend/eslint/eslint-enforced-rules.md

```diff
@@ -0,0 +1,101 @@
+### ESLint Audit
```

Comment:
What would you say the goal of this document is? Should it be always kept up-to-date? Does it represent a snapshot in time? How can a future engineer can benefit from this?

Reply:
I'll add an objective for this. Also, I would love to keep this up-to-date but I think it'll be too much work to do unless there is an assigned person for that. For the time being, I think I'm going to keep it as a snapshot in time.